modules/ml/src/main/java/org/apache/ignite/ml/DatasetTrainer.java

```diff
@@ -23,20 +23,20 @@
 /**
```

Comment:
MB move this and MultiLabelDatasetTrainer with SingleLabelDatasetTrainer into trainers package?